src/components/BuildStatus/BuildStatus.tsx

```diff
@@ -26,8 +28,10 @@ interface IBuildStatus {
  * @param long - Whether the component should be of the long version
  * @param includeBuildLink - Whether the build name should be a link to the build page
  * @param includeConfigLink - Whether the build config (in the long version) should be a link to the build config page
+ * @param hideDatetime - Whether build datetime will be displayed
+ * @param hideUsername - Whether build username will be displayed
  */
-export const BuildStatus = ({ build, long, includeBuildLink, includeConfigLink }: IBuildStatus) => {
+export const BuildStatus = ({ build, long, includeBuildLink, includeConfigLink, hideDatetime, hideUsername }: IBuildStatus) => {
```

Comment:
maybe we should use "show..." instead of "hide..."? and set it to false as default if necessary.

Reply:
I'd like to display those properties ideally everywhere, it should be hidden only in some specific like page title, but we can revalidate it again later if we change our mind.